spec/requests/api/policies_spec.rb

```diff
@@ -178,6 +178,27 @@ def test_policy_profile_query(object, object_policy_profiles_url)
       )
       expect_result_resources_to_include_data("policies", "guid" => p_guids)
     end
+
+    it "adds Policies to a Policy Profile" do
+      api_basic_authorize
+
+      run_post("#{policy_profiles_url(ps2.id)}/policies", gen_request(:assign, [{"id" => p1.id}, {"id" => p2.id}]))
+
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body["results"].count).to eq(2)
+      expect(ps2.reload.miq_policies.count).to eq(3)
+      expect(response.parsed_body["results"].first["guid"]).to eq(p1.guid)
+      expect(response.parsed_body["results"].second["guid"]).to eq(p2.guid)
+    end
+
+    it "removes a Policy from a Policy Profile" do
+      api_basic_authorize
+
+      run_post("#{policy_profiles_url(ps2.id)}/policies", gen_request(:unassign, [{"id" => p3.id}]))
+
+      expect(response).to have_http_status(:ok)
+      expect(ps2.reload.miq_policies.count).to eq(0)
+    end
```

Comment:
Done, thanks!

Notice I added `object.reload` in the unassign spec, without it the test fails.